7.application.md

```diff
@@ -31,116 +31,94 @@ This section defines the instances of components to create with the application.
     version: v1.0.0
-    description: "Description of this deployment"
+    description: "Brief description of the application"
 spec:
   components:
-    - name: my-web-app-component
-      type: Server
-      settings:
-        containers:
-        - name: my-cool-workload
-          image: example/very-cool-workload:0.1.2@sha256:verytrustworthyhash
-          resources:
-            cpu:
-              required: 1.0
-            memory:
-              required: 100MB
-          cmd:
-          - "bash lscpu"
-          ports:
-          - name: http
-            value: 8080
+    - name: publicweb
+      type: web-ui
+      properties: # properties targeting component parameters.
+        image: example/web-ui:v1.0.2@sha256:verytrustworthyhash
+        param_1: "enabled" # param_1 is defined on the web-ui component
+      traits:
+        - type: ingress # ingress trait providing a public endpoint for the publicweb component of the application.
+          properties: # properties are defined by the trait CRD spec. This example assumes path and port.
+            path: /
+            port: 8080
+      scopes:
+        "healthscopes.core.oam.dev": "app-health" # An application level health scope including both components.
+    - name: backend
+      type: company/test-backend # test-backend is referenced from other namespace
+      properties:
+        debug: "true" # debug is a parameter defined in the test-backend component.
       traits:
-        - name: scaler
+        - type: scaler # scaler trait to specify the number of replicas for the backend component
           properties:
-            replicas: 2
+            replicas: 4
       scopes:
-        - scopeRef:
-            apiVersion: core.oam.dev/v1alpha2
-            kind: NetworkScope
-            name: example-vpc-network
+        "healthscopes.core.oam.dev": "app-health" # An application level health scope including both components.
 ```
 
-The example above illustrates a complete application, including its scopes, components and their traits. This application assumes a [WorkloadDefinition](4.workload_types.md) named `Server` and a [TraitDefinition](6.traits.md) named `scaler` are present in the runtime cluster. The templates defined in them will enforce the schema of this application.
+The example above illustrates a complete application, including its scopes, components and their traits. This application assumes the existence of two [Components](3.component.md) representing the web user interface and the backend of an imaginary application. Two [Trait](6.traits.md) are used to augment the functionality of the components. First an scaler on the backend to set the replication factor, and an Ingress one to expose the application to the outside world. Additionally, a HealthScope is used to have a common point to check the status of both the web and backend components. This is an arbitrary set of components for the purpose of the example and not necessarilly requires all of them.
```

Comment:
The references to traits scaler and Ingress are inconsistent.  Should they both be capitalized or not?
Also just use "...and in ingress to expose...".

Reply:
fixed